tests/Query/BuilderTest.php

```diff
@@ -433,6 +434,62 @@ function (Builder $builder) {
                 ->orWhereNotBetween('id', collect([3, 4])),
         ];
 
+        yield 'where like' => [
+            ['find' => [['name' => new Regex('^acme$', '')], []]],
+            fn (Builder $builder) => $builder->where('name', 'like', 'acme'),
+        ];
+
+        yield 'where like escape' => [
+            ['find' => [['name' => new Regex('^\^acme\$$', '')], []]],
+            fn (Builder $builder) => $builder->where('name', 'like', '^acme$'),
+        ];
+
+        yield 'where like %' => [
+            ['find' => [['name' => new Regex('.*acme.*', '')], []]],
+            fn (Builder $builder) => $builder->where('name', 'like', '%acme%'),
```

Comment:
Both the [PostgreSQL](https://www.postgresql.org/docs/current/functions-matching.html#FUNCTIONS-LIKE) and [MySQL](https://dev.mysql.com/doc/refman/8.0/en/string-comparison-functions.html#operator_like) docs are consistent about the `%` and `_` operators:

> An underscore (_) in pattern stands for (matches) any single character; a percent sign (%) matches any sequence of zero or more characters.

I think we're missing a test that `_` is concerted to `?` in a regex pattern.

Also, both of those operators can be escaped with a backslash, so you might want to test _that_ as well to demonstrate that we don't inadvertently build a regex pattern with `.*` or `?`.

Reply:
Added support for both `%` to `.*` and `_` to `.` (not `?` which is for 0 or 1 instance of the preceding pattern).
Also fixed unescaping `\%` and `\_`.